solr/core/src/test/org/apache/solr/security/BasicAuthOnSingleNodeTest.java

```diff
@@ -45,13 +45,13 @@ public void setupCluster() throws Exception {
         .setBasicAuthCredentials("solr", "solr")
         .process(cluster.getSolrClient());
     cluster.waitForActiveCollection(COLLECTION, 4, 4);
+  }
 
-    JettySolrRunner jetty = cluster.getJettySolrRunner(0);
```

Comment:
I removed this part of initialization - don't see why we should tear down and recreate cluster every time?